From e402ad04d1e2a8328f293ab2e8298354389893f8 Mon Sep 17 00:00:00 2001
From: Angela Czubak <aczubak@marvell.com>
Date: Wed, 29 May 2019 15:21:53 +0200
Subject: [PATCH 288/386] octeontx: increment MBOX major version

The MBOX did not change, however, the semantics did.
Since the decision to keep all devices open in ODP and perform
domain reset on any device closure in kernel we want to ensure that
someone does not run older ODP, which may close devices on a whim,
with newer kernel, which performs domain reset more often.

Change-Id: I723ae0c303e8a30782eff6d951699d96b049e80e
Signed-off-by: Angela Czubak <aczubak@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/10194
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Reviewed-by: Stanislaw Kardach <Stanislaw.Kardach@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 drivers/net/ethernet/cavium/octeontx-83xx/octeontx_mbox.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/cavium/octeontx-83xx/octeontx_mbox.h b/drivers/net/ethernet/cavium/octeontx-83xx/octeontx_mbox.h
index 7b3df3ac682a..94b837428f25 100644
--- a/drivers/net/ethernet/cavium/octeontx-83xx/octeontx_mbox.h
+++ b/drivers/net/ethernet/cavium/octeontx-83xx/octeontx_mbox.h
@@ -269,7 +269,7 @@ struct mbox_intf_ver {
 
 static const struct mbox_intf_ver MBOX_INTERFACE_VERSION = {
 	.platform = 0x01,
-	.major = 0x00,
+	.major = 0x01,
 	.minor = 0x01
 };
 
-- 
2.17.1

